### ISSUE TITLE
Allow linebreaks (or other way to delimit exactives) in message

### ISSUE THREAD

**Issue by tadhg-ohiggins** [fugacious/fugacious#60]
Let's say I'm trying to tell someone their password is `"insecure"?:<>-*&^`. Right now, there's no intuitive way to delimit the password, and they'd see something like:

 `Your password is "insecure"?:<>-*&^ and you should change it soon.`

I think the easiest delimiter would be blank lines, so that the message could be:
```
Your password is:

"insecure"?:<>-*&^

and you should change it soon.
```

Explicit support for ways to mark elements as code would fix this too, but I assume they'd be harder.

**Comment by rogeruiz** [comment]
@tadhg-ohiggins maybe markdown processing of the content coming in the field along with a rich markdown editor similar to Github's? 💭

**Comment by rdpiazza** [comment]
Want to add a +1 to this ticket.  I started to use Fugacious to send a message last night, but it was unusable once the line returns were stripped.  I ended up needing to exchange the info through a long tedious phone call instead.

**Comment by carlmjohnson** [comment]
I made a PR that fixes this, #69.

**Comment by rdpiazza** [comment]
Hooray! Thanks for adding this feature!
